aten/src/ATen/common_with_cwrap.py

```diff
@@ -92,9 +92,9 @@ def signature(option, kwarg_only_count):
 
 
 def sort_by_number_of_options(declaration, reverse=True):
```

Comment:
nit: rename this to `sort_by_number_of_args` or something similar. `options` doesn't mean anything here anymore